src/pages/FarmV3/FarmHeader.tsx

```diff
@@ -223,145 +174,63 @@ export const FarmHeader: FC = () => {
+              &nbsp;
+              <IconTooltip tooltipType={'outline'}>{getTooltipText(userPubKey ? index : index + 1)}</IconTooltip>
+            </ContainerTitle>
+            <h2> {card.value}</h2>
+          </Container>
+        ))}
       </div>
-      <POOL_CARD_WRAPPER>
-        {allPoolDataWithApy?.length
-          ? allPoolDataWithApy
-              .sort((a, b) => b.apy - a.apy)
-              ?.slice(0, 4)
-              ?.map((card) => (
-                <POOL_CARD key={card?.token}>
-                  <div tw="flex flex-row justify-center items-center mb-3.5 sm:mb-2 sm:justify-between">
-                    <img
-                      src={`/img/crypto/${card.token}.svg`}
-                      alt="pool-icon"
-                      height={40}
-                      width={40}
-                      tw="mr-2.5"
-                    />
-                    <h2 tw="text-lg font-semibold text-black-4 dark:text-grey-5 mr-auto sm:text-average">
-                      {card?.token}
-                    </h2>
-                    <div
-                      tw="flex flex-row h-[30px] w-[110px] flex flex-row justify-center items-center
-                      rounded-circle dark:bg-black-2 bg-grey-4 sm:w-[100px]"
-                    >
-                      <img
-                        src={`/img/assets/${
-                          card.assetType === 1 ? 'Primary' : card.assetType === 2 ? 'Hyper' : 'Stable'
-                        }_pools_${mode}.svg`}
-                        alt="pool-type"
-                        tw="mr-1.25 h-[21px] w-[19px]"
-                      />
-                      <h4
-                        tw="text-lg font-semibold text-black-4 dark:text-white 
-                        text-regular font-semibold"
-                      >
-                        {card.assetType === 1 ? 'Primary' : card.assetType === 2 ? 'Hyper' : 'Stable'}
-                      </h4>
-                    </div>
-                  </div>
-                  <div tw="flex items-center leading-[22px] sm:mt-3.5">
-                    <h4 tw="text-grey-1 text-regular font-semibold dark:text-grey-2">APY: </h4>
-                    <h4 tw="text-black-4 text-regular font-semibold dark:text-grey-5 ml-1">
-                      {card?.apy ? `${card.apy?.toFixed(2)}%` : '00.00%'}
-                    </h4>
-                  </div>
-                </POOL_CARD>
-              ))
-          : [0, 1, 2, 3].map((item) => (
-              <SkeletonCommon
-                height="97px"
-                width={!checkMobile() ? '95%' : '257px'}
-                key={item}
-                style={checkMobile() && { marginRight: '15px' }}
-              />
-            ))}
-          </POOL_CARD_WRAPPER> */}
-    </>
+      <Button
+        className="cursor-pointer ml-auto p-0 min-w-7.5"
+        variant={'ghost'}
+        onClick={() => setPoolSelectionModal(true)}
```

Comment:
This one should be in useCallback wrapper

Reply:
True,  substituted for built in useBoolean hook.